utils/hct/hctdb.py

```diff
@@ -5848,9 +5848,97 @@ def UFI(name, **mappings):
         )
         next_op_idx += 1
 
-        # End of DXIL 1.9 opcodes.
+        self.add_dxil_op(
+              "MatVecMul",
+              next_op_idx,
+              "MatVecMul",
+              "Matrix-Vector Multiply",
+              "<hfwi,<hfwi",
+              "",
+              [
+                  db_dxil_param(0, "$x0", "outputVector", "output vector"),
+                  db_dxil_param(2, "$x1", "inputVector", "input vector"),
+                  db_dxil_param(3, "i1", "isInputSigned", "input signed op kind"),
```

Comment:
```suggestion
                  db_dxil_param(3, "i1", "isInputSigned", "is input signed"),
```

This is repeated for other ops, also need to update the isOutputSigned descriptions as well.

Reply:
Changed param names and descriptions as suggested (could not apply from github). Resolved.